appendices/VK_FOOL_printed_surface.txt

```diff
@@ -0,0 +1,206 @@
+              a satisfactory solution to this question. A future maintenance
+              extension may be necessary to address this concern.
+
+
+9) Is elongated swapchain backpressure from image acquisition a concern for
+    the practicality of apps wishing to implement this extension?
+
+*RESOLVED*: Yes; please adjust expectations accordingly.
+
+
+10) How will duplex printing be handled?
+
+*RESOLVED*: Duplex printing support can be queried via
+            flink:vkGetPrinterPropertiesFOOL and enabled via
+            tname:VkPrintedSurfaceCreateInfoFOOL::pname:enableDuplex.
+
+
+11) Should this extension specify features available in high-end printers such as autmatic stapling and hole punching?
+
+*RESOLVED*: Yes. These must: not be required, however they are prevalent enough
+            that they can not be overlooked by this extension.
+
+
+12) Will priority in the print queue be handled by this extension?
+
+*RESOLVED*: Please wait for the VK_FOOL_print_queue_priority extension.
+
+
+13) Will this extension support printing "zoom levels"? Possibly allowing multiple presented images to appear on a single page?
+
+*RESOLVED*: No, we are printing frames here, not slides.
+
+
+14) Are print color profiles covered under the scope of this extension?
+
+*RESOLVED*: No.
+
+
+15) Why are you doing this?
+
+*UNRESOLVED*
+
+
+16) How does this extension interact with VK_EXT_custom_border_color?
+
+*RESOLVED*: It does not. VK_EXT_custom_border_color is in reference to
+            border sampling, not print borders. A future extension could
+            potentially handle custom printed border colors.
+
+
+17) How does this extension interact with overlay layers (such as MangoHUD)?
+
+*UNRESOLVED*: The API specification for this extension is not the place for
+              such interactions to be considered. Individual layers (such as
+              MangoHUD) have the ability to change their behavior based on the
+              surface being presented to.
+
+18) How will this extension interact with HDR images?
+
+*RESOLVED*: High Dynamic Range printing will not be supported by this extension.
+
+
+19) Is this a serious extension?
+
+*RESOLVED*: No. Why? Do you want it to be?
+
+
+20) Will this extension allow for enumeration of printers (similar to VK_KHR_display), or leave enumeration up to the application and use OS-API native handles (similar to other WSI surface extensions)?
+
+*RESOLVED*: In order to provide a consistent development experience, a printer
+            enumeration API will be provided.
+
+
+21) Will the extension support tearing?
+
+*RESOLVED*: No. The printed surface will only support perfect prints. Tearing is left to the end-user.
+
+
+22) What vendor string should be used for this extension?
+
+*RESOLVED*: FOOL; as to create a very clear note that this is an April Fools
+            joke and not affiliated in any way with our day-to-day work in the
+            Vulkan ecosystem through our employeers.
```

Comment:
![PrintInc](https://user-images.githubusercontent.com/3791331/161291543-2fda0d36-6a28-4124-b816-1db71d2a279c.PNG)

Reply:
> ![PrintInc](https://user-images.githubusercontent.com/3791331/161291543-2fda0d36-6a28-4124-b816-1db71d2a279c.PNG)

When I worked at SGI, we had an online tool to browse the org chart. At one point I discovered that the deepest reporting chain in the company was about 13 people deep, consisting of one hapless person on the bottom who was the sole employee of their manager, going up in this same fashion about 6 levels more inside SGI Japan, then turning into a more normal set of relationships up to the CEO.